docs/_templates_overwrite/index.html

```diff
@@ -1,568 +1,628 @@
+                <div class="col-lg-3 col-md-6">
+                    <div class="product-single text-center">
+                        <div class="prod-details">
+                            <p>
+                            <a href="https://github.com/anki-code/xontrib-pipeliner" target="_blank">pipeliner</a>
+                            </p>
+                            <p>Let your pipe lines from the standard output flow thru the Python code in the xonsh shell. </p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-lg-3 col-md-6">
+                    <div class="product-single text-center">
+                        <div class="prod-details">
+                            <p>
+                            <a href="https://github.com/anki-code/xontrib-output-search" target="_blank">output-search</a>
+                            </p>
+                            <p>Get identifiers, paths, URLs and words from the previous command output and use them for the next command in the xonsh shell. </p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-lg-3 col-md-6">
+                    <div class="product-single text-center">
+                        <div class="prod-details">
+                            <p>
+                            <a href="https://github.com/anki-code/xontrib-back2dir" target="_blank">back2dir</a>
+                            </p>
+                            <p>Return to the most recently used directory when starting the xonsh shell.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-lg-3 col-md-6">
+                    <div class="product-single text-center">
+                        <div class="prod-details">
+                            <p>
+                            <a href="https://github.com/anki-code/xontrib-argcomplete" target="_blank">argcomplete</a>
+                            </p>
+                            <p>Argcomplete support to tab completion of python and xonsh scripts in the xonsh shell. </p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-lg-12">
+                    <div class="load-more-btn text-center">
+                        <a href="https://github.com/topics/xontrib" target="_blank"><i class="icofont-download"></i> More xontribs →</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="blog-area" data-scroll-index="5">
+        <div class="container">
+            <div class="col-lg-6 offset-lg-3 col-md-8 offset-md-2">
+                <div class="section-heading text-center">
+                    <h2>Xonsh News</h2>
+                    <p>Most interesting publications around xonsh.</p>
+                </div>
+            </div>
```

Comment:
I'm worried about this section bit-rotting.  Maybe we can keep it to "Talks about xonsh?"